systest/Makefile

```diff
@@ -102,6 +90,7 @@ export FROM_AGENT_SESSION := from.agent_$(SUBJECTCODE_ID)_$(TIMESTAMP)
 
 microservice_setup_tlc_session:
 	@echo "setting up TLC session..."
+	sudo -E chown -Rf jenkins:jenkins /home/jenkins && \
```

Comment:
Can we specify this chown for our current build target? Instead of chowning /home/jenkins